### PR TITLE
fix: Add workspace field in sign-up form

### PR DESCRIPTION
**Pull request by killbond007**
## Description

Add workspace field in sign-up form

Capture d’écran 2021-03-17 à 22 33 44

## Links

- https://github.com/stargately/launch-room/issues/57

### CODE REVIEW

File: translations/en.yaml [hunk 1]
```diff
@@ -1,3 +1,5 @@
+auth.sign_up.label.workspace: Workspace Name
```
Comment (bitQ2019) [inline translations/en.yaml:1]: why this long 😃

File: src/shared/onefx-auth-provider/email-password-identity-provider/email-password-identity-provider-handler.tsx [hunk 2]
```diff
@@ -141,6 +142,18 @@ export function setEmailPasswordIdentityProviderRoutes(server: MyServer): void {
...
+
+        const userWorkspace = await server.model.workspaceModel.create({
+          name: workspaceName,
+          owner: user._id,
+        });
+
+        await server.model.userWorkspace.create({
+          workspace: userWorkspace._id,
+          user: user._id,
+          role: WorkspaceRole.follower,
```
Comment (bitQ2019) [inline src/shared/onefx-auth-provider/email-password-identity-provider/email-password-identity-provider-handler.tsx:154]: the owner is a admin I think...

**Review by bitQ2019 — CHANGES_REQUESTED** [review]

File: translations/en.yaml (hunk 1 above)
Comment (killbond007) [reply translations/en.yaml:1]: auth.sign_up.label_workspace ? :)

File: src/shared/onefx-auth-provider/email-password-identity-provider/email-password-identity-provider-handler.tsx (hunk 2 above)
Comment (killbond007) [reply src/shared/onefx-auth-provider/email-password-identity-provider/email-password-identity-provider-handler.tsx:154]: Tian says use "owner" , but there is not "owner" in workspaceRole . "owner" is "admin" ? @puncsky

File: translations/en.yaml (hunk 1 above)
Comment (bitQ2019) [reply translations/en.yaml:1]: maybe two-layer is enough?  in my simple opinion...

Comment (killbond007) [reply translations/en.yaml:1]: yes sign_up is not necessary

**Pull request merged**